modules/administration_manual/pages/installation/manual_installation_ubuntu_16_04.adoc

```diff
@@ -0,0 +1,163 @@
+= Manual Installation on Linux - Ubuntu 16.04
+:nginx-app-tracing-url: https://www.nginx.com/blog/application-tracing-nginx-plus/
+
+== Introduction
+This guide describes how to install the required dependencies, the database and ownCloud itself in a
+single server setup.
+
+
+[[install-the-required-packages]]
+== Prepare your Server
+
+*Necessary steps to prepare the installation of ownCloud*
+
+Before we can start to install ownCloud, we need to make sure that everything needed
+is available.
+
+Steps to take:
+
+* Install MariaDB Database
+* Install Redis Server
+* Install Apache and the necessary dependencies
+* Configure Apache
+* Download and install ownCloud 
+
+
+=== Install MariaDB Database
+
+Perform the following steps to install MariaDB and setting up a Database for ownCloud.
+
+....
+apt update
+apt install -y mariadb-server-10.0
+....
+After this has finish, create the database and a database user for ownCloud to use:
+
+....
+mysql -u root
+
+MariaDB [(none)]> create database owncloud;
+MariaDB [(none)]> GRANT USAGE ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> GRANT ALL PRIVILEGES ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> flush privileges;
+MariaDB [(none)]> exit
+....
+
+=== Install Redis Server
+Redis in needed for Transactional File Locking in ownCloud. Perform the following step to install it:
+....
+apt install -y redis-server
+....
+
+=== Install Apache and the necessary dependencies
+The following steps will install the Apache Webserver and the dependencies needed for ownCloud:
+....
+apt install -y \
```

Comment:
I strongly suggest formatting this command example differently, so that it's more compact, yet still readable. For example:

```
apt install -y \
    apache2 gettext-base && libapache2-mod-php \
    php-apcu php-curl php-gd php-imagick \
    php-intl php-json php-ldap php-mbstring \
    php-mcrypt php-mysql php-redis php-smbclient \
    php-soap php-xml php-zip smbclient && \
a2enmod ssl rewrite headers env dir mime ldap && \
service apache2 restart
```